lib/dune

```diff
@@ -4,7 +4,7 @@
   (libraries ctypes ctypes.foreign str)
   (c_flags         (:include c_flags.sexp))
   (c_library_flags (:include c_library_flags.sexp))
-  (ocamlopt_flags (-ccopt ("-Wl,-no-as-needed")))
+  (ocamlopt_flags (-ccopt ("-Wl")))
```

Comment:
If you want to remove this option, you have to remove the whole line? But if I remember well this is needed on the Ubuntu system. But it could be great if you try to remove this line, we will see if the tests in TravisCI work fine without it.

Reply:
Is there an easy way to add conditionals in the stanzas?

Reply:
Configurator script is the way to go. On it.